test/core_spec.js

```diff
@@ -220,4 +220,53 @@ describe('application logic', () => {
       }));
     });
   });
+
+  describe('addMessage', () => {
+    it('adds a message to chat with user and timestamp', () => {
+      let user = "cat";
+      let message = "meow";
```

Comment:
most style guides prefer const where possible, to promote maximum strictness and prevent accidental modification